packages/react/src/components/TreeView/TreeNode.tsx

```diff
@@ -104,6 +117,8 @@ export type TreeNodeProps = {
   /**
    * **Note:** this is controlled by the parent TreeView component, do not set manually.
    * Array containing all selected node IDs in the tree
+   *     @deprecated The `selected` prop for `TreeNode` has
+   *     been deprecated after the introduction of context. It will be removed in the next major release.
```

Comment:
```suggestion
   * @deprecated The `selected` prop for `TreeNode` has
   * been deprecated after the introduction of context. It will be removed in the next major release.
```